newsfragments/3537.misc.rst

```diff
@@ -0,0 +1 @@
+Recovery devices can now be renamed when created.
```

Comment:
Shouldn't this be a feature instead of miscellaneous ?

Reply:
What's the other part ?

Reply:
The device recovery is the feature, this change is just a miscellaneous one on it.